editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -67,30 +61,12 @@ const DRAG_START_THRESHOLD = 2
 
 export function isResizing(editorState: EditorState): boolean {
   // TODO retire isResizing and replace with isInteractionActive once we have the strategies turned on, and the old controls removed
```

Comment:
Given this function and the one below are both returning true in exactly the same conditions, it looks like it's time for this `TODO` (maybe as a follow up PR if it causes too much fallout in this one?)